mne_qt_browser/_pg_figure.py

```diff
@@ -1850,6 +1857,26 @@ def _value_changed(self, new_value, value_name):
     def _toggle_antialiasing(self, _):
         self.weakmain()._toggle_antialiasing()
 
+    def _update_spinbox_values(self, *args, **kwargs):
+        """Update spinbox values."""
+        scaler = 1 if self.mne.butterfly else 2
+        if len(args) > 0:
+            new_value = args[0]
+            ch_type = kwargs["ch_type"]
+            self.mne.scalings[ch_type] = new_value / (
+                self.mne.unit_scalings[ch_type] * scaler / self.mne.scale_factor
```

Comment:
... and this could use the same helper but with some divisions probably?

Reply:
Added helper in 5eea6c6038e49fc44bf4dc85c3779f22c91d83c3